instances/azp-nano-asg/init.sh.tpl

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+function terminate {
+    # Terminate instances in 1 min
+    shutdown -h +1
```

Comment:
(optional) Should we give ourselves a bit more debugging time here? Say 3 min?

Reply:
i would say probably no - we want these instances to clear up quickly